src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String FIRST_OPERATION_NAME = "supply";
+    private static final String SECOND_OPERATION_NAME = "buy";
+    private static final String DELIMITER = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] textLines = getInputText(fromFileName).split(System.lineSeparator());
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        for (String line : textLines) {
+            String[] lineInfo = line.split(DELIMITER);
+            if (lineInfo[0].equals(FIRST_OPERATION_NAME)) {
```

Comment:
let's also make constants for word's position (operation,amount)

Reply:
done